### ISSUE TITLE
Performance Tests often fail to start

### ISSUE THREAD

**Issue by heifner** [AntelopeIO/leap#889]
It is unclear what is causing the failure. Here is the log output for the node that doesn't start. This is a fresh chain, unclear why it is even attempting to read from a block file. It looks like maybe it is trying to read from an empty block log file. Maybe we have some race on when the block log is created.

```
info  2023-03-24T16:58:46.990 nodeos    protocol_feature_manag:930    operator()           ] Saved default specification for builtin protocol feature 'PREACTIVATE_FEATURE' (with digest of '0ec7e080177b2c02b278d5088611686b49d739925a92d9bfcacd7fc6b74053bd') to: /home/heifner/ext/leap/cmake-build-debug/etc/eosio/node_00/protocol_features/BUILTIN-PREACTIVATE_FEATURE.json
warn  2023-03-24T16:58:46.991 nodeos    chain_plugin.cpp:1083         plugin_initialize    ] 13 NSt8ios_base7failureB5cxx11E: cfile: /home/heifner/ext/leap/cmake-build-debug/TestLogs/performance_test583202/node_00/blocks/blocks.log unable to read 4 bytes; only read 0, eof: true, ferror: 0: iostream error
rethrow cfile: /home/heifner/ext/leap/cmake-build-debug/TestLogs/performance_test583202/node_00/blocks/blocks.log unable to read 4 bytes; only read 0, eof: true, ferror: 0: iostream error: 
    {"what":"cfile: /home/heifner/ext/leap/cmake-build-debug/TestLogs/performance_test583202/node_00/blocks/blocks.log unable to read 4 bytes; only read 0, eof: true, ferror: 0: iostream error"}
    nodeos  chain_plugin.cpp:1083 plugin_initialize

error 2023-03-24T16:58:46.991 nodeos    main.cpp:161                  main                 ] 13 NSt8ios_base7failureB5cxx11E: cfile: /home/heifner/ext/leap/cmake-build-debug/TestLogs/performance_test583202/node_00/blocks/blocks.log unable to read 4 bytes; only read 0, eof: true, ferror: 0: iostream error
rethrow cfile: /home/heifner/ext/leap/cmake-build-debug/TestLogs/performance_test583202/node_00/blocks/blocks.log unable to read 4 bytes; only read 0, eof: true, ferror: 0: iostream error: 
    {"what":"cfile: /home/heifner/ext/leap/cmake-build-debug/TestLogs/performance_test583202/node_00/blocks/blocks.log unable to read 4 bytes; only read 0, eof: true, ferror: 0: iostream error"}
    nodeos  chain_plugin.cpp:1083 plugin_initialize
```

**Comment by heifner** [comment]
When it fails the blocks.log file exists at time chain_plugin plugin_initialize 
```
info  2023-03-24T17:39:36.010 nodeos    chain_plugin.cpp:470          plugin_initialize    ] initializing chain plugin
info  2023-03-24T17:39:36.010 nodeos    chain_plugin.cpp:534          plugin_initialize    ] blocks-dir: blocks
info  2023-03-24T17:39:36.010 nodeos    chain_plugin.cpp:539          plugin_initialize    ] my->blocks-dir: /home/heifner/ext/leap/cmake-build-debug/TestLogs/performance_test590182/node_00/blocks
info  2023-03-24T17:39:36.010 nodeos    chain_plugin.cpp:541          plugin_initialize    ] exists: true
```

**Comment by heifner** [comment]
Resolved by #926
